wolfssl/wolfcrypt/wc_port.h

```diff
@@ -1264,7 +1264,11 @@ WOLFSSL_ABI WOLFSSL_API int wolfCrypt_Cleanup(void);
 #endif
 
 #ifndef WOLFSSL_NO_FENCE
-    #if defined (__i386__) || defined(__x86_64__)
+    #ifdef XFENCE
+        /* use user-supplied XFENCE definition. */
```

Comment:
Will never reach this line 
`!defined(WOLFSSL_NO_FENCE) && defined(WOLFSSL_NO_FENCE)`